packages/client/modules/userDashboard/containers/OrgAuthentication/OrgAuthenticationRoot.tsx

```diff
@@ -1,10 +1,21 @@
 import React, {Suspense} from 'react'
+import useQueryLoaderNow from '../../../../hooks/useQueryLoaderNow'
+import {renderLoader} from '../../../../utils/relay/renderLoader'
+import orgAuthenticationQuery, {
+  OrgAuthenticationQuery
+} from '../../../../__generated__/OrgAuthenticationQuery.graphql'
 import OrgAuthentication from '../../components/OrgAuthentication/OrgAuthentication'
 
-const OrgAuthenticationRoot = () => {
+interface Props {
+  orgId: string
+}
+
+const OrgAuthenticationRoot = (props: Props) => {
+  const {orgId} = props
+  const queryRef = useQueryLoaderNow<OrgAuthenticationQuery>(orgAuthenticationQuery, {orgId})
   return (
-    <Suspense fallback={''}>
-      <OrgAuthentication />
+    <Suspense fallback={renderLoader({Loader: <div />})}>
```

Comment:
+1 no need for renderLoader call here, what error were you seeing with just rendering an empty string?